sdk/src/camera_96tof1.cpp

```diff
@@ -59,11 +66,13 @@ aditof::Status Camera96Tof1::setMode(const std::string &mode,
     using namespace aditof;
     Status status = Status::OK;
 
-    LOG(INFO) << "Chosen mode: " << mode.c_str();
-
-    std::vector<struct rangeStruct> rangeValues = {
-        {"near", 250, 800}, {"medium", 300, 4500}, {"far", 3000, 6000}};
+    // Set the values specific to the Revision requested
+    auto cam96tof1Specifics =
+        std::dynamic_pointer_cast<Camera96Tof1Specifics>(m_specifics);
+    Revision revision = cam96tof1Specifics->getRevision();
+    m_rangeValues = RangeValuesForRevision.at(static_cast<int>(revision));
```

Comment:
Maybe use a map here to store the range values for revision

Reply:
done. thanks